ios/tools/prepRelease.sh

```diff
@@ -50,12 +50,10 @@ mkdir -p "${UPLOAD_DIR}"
 echo "Writing changelog to $CHANGELOG_PATH"
 get_version_notes "ios" "${BUILD_NUMBER}" "$KEYMAN_TIER" > $CHANGELOG_PATH
 echo "* Minor fixes and performance improvements" >> $CHANGELOG_PATH
-assertFileExists "${CHANGELOG_PATH}"
```

Comment:
These assertions seem unnecessary given previous line is writing to the file anyway